### PR TITLE
[DE] Add `Past osu!store items`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
- [x] Check if #12388 makes any changes in `Past osu!store items`
- [x] Translate `hit judgements`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Past_osu!store_items/de.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+tags:
+  - store
+  - merchandise
+  - legacy
+  - Shop
+---
+
+# Vergangene Artikel aus dem osu!store
+
+In diesem Artikel werden verschiedene Produkte gezeigt, die mal im [osu!store](https://osu.ppy.sh/store/listing) verfügbar waren, mittlerweile aber nicht mehr produziert werden und auch nicht zum Verkauf stehen.
```
Comment (The-Last-Cookie) [inline wiki/Past_osu!store_items/de.md:11]:
```suggestion
Auf dieser Seite werden verschiedene Produkte gezeigt, die mal im [osu!store](https://osu.ppy.sh/store/listing) verfügbar waren, mittlerweile aber nicht mehr produziert werden und auch nicht zum Verkauf stehen.
```

File: wiki/Past_osu!store_items/de.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# Vergangene Artikel aus dem osu!store
+
+Auf dieser Seite werden verschiedene Produkte gezeigt, die mal im [osu!store](https://osu.ppy.sh/store/listing) verfügbar waren, mittlerweile aber nicht mehr produziert werden und auch nicht zum Verkauf stehen.
+
+## osu! goods
+
+![Storeseite von osu! goods CafePress](img/osu-goods.png)
+
+**osu! goods** war eine Kollektion von offiziellen osu!-Markenartikeln, die über den Onlinehändler [CafePress](https://en.wikipedia.org/wiki/CafePress) zwischen 2009 und 2012 verkauft wurden. Alle Artikel wurden von CafePress gedruckt und produziert, wobei ein Teil der Einnahmen in die [Spielentwicklung](https://osu.ppy.sh/store/listing) flossen.
```
Comment (FreezyLemon) [inline wiki/Past_osu!store_items/de.md:17]: The link for "supporting the game" is a bit unfortunate in that position

Maybe

```suggestion
**osu! goods** war eine Kollektion von offiziellen osu!-Markenartikeln, die über den Onlinehändler [CafePress](https://en.wikipedia.org/wiki/CafePress) zwischen 2009 und 2012 verkauft wurden. Alle Artikel wurden von CafePress gedruckt und produziert, wobei ein Teil der Einnahmen [in die Spielentwicklung flossen](https://osu.ppy.sh/store/listing).
```

or alternatively

```suggestion
**osu! goods** war eine Kollektion von offiziellen osu!-Markenartikeln, die über den Onlinehändler [CafePress](https://en.wikipedia.org/wiki/CafePress) zwischen 2009 und 2012 verkauft wurden. Alle Artikel wurden von CafePress gedruckt und produziert. Ein Teil der Einnahmen wurde zur [Unterstützung der Spielentwicklung](https://osu.ppy.sh/store/listing) verwendet.
```

File: wiki/Past_osu!store_items/de.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+- [osu! Large Stickers 40-Pack](https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume.288841446)
+
+## osu!tablet
+
+![osu! tablet](img/osu-tablet.jpg)
+
+*Siehe auch: [osu!tablet official thread](https://osu.ppy.sh/community/forums/topics/169139)*\
+*Für Hilfe zu diesem Produkt, siehe: [Shoparchiv § osu!tablet](/wiki/Help_centre/Store/Store_archive#osu-tablet)*
+
+Das **osu!tablet** war ein von osu! offiziell vertriebenes Grafiktablett, das in Zusammenarbeit mit der Elektronikfirma [HUION](https://www.huion.com/) entstand. Es wurde von ::{ flag=MY }:: [flyte](https://osu.ppy.sh/users/3103765) entworfen und als eine erschwingliche Alternative für die, die einfach ein Grafiktablett für osu! haben wollten, herausgebracht.
```
Comment (FreezyLemon) [inline wiki/Past_osu!store_items/de.md:75]: It was an official tablet being sold, not a tablet being sold officially :P

```suggestion
Das **osu!tablet** war ein offizielles, von osu! vertriebenes Grafiktablett, das in Zusammenarbeit mit der Elektronikfirma [HUION](https://www.huion.com/) entstand. Es wurde von ::{ flag=MY }:: [flyte](https://osu.ppy.sh/users/3103765) entworfen und als eine erschwingliche Alternative für die, die einfach ein Grafiktablett für osu! haben wollten, herausgebracht.
```

File: wiki/Past_osu!store_items/de.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Das Produkt wurde in zwei Ausführungen angeboten: Das originale "[osu!tablet v1](https://www.youtube.com/watch?v=27RkPY5lWBw)", welches zuerst 2013 hergestellt wurde. Das verbesserte [osu!tablet v2](/wiki/Guides/Tablet_purchase#osu!store) kam ab 2016 auf den Markt, bevor es im Jahre 2017 im Rahmen einer [Anpassung der Entwicklungsschwerpunkte](https://twitter.com/ppy/status/846190076853870592) eingestellt wurde.
+
+## osu! beatmap blueprints
+
+![osu! beatmap blueprints](img/osu-beatmap-blueprints.jpg)
+
+*Für den Newsbeitrag, siehe: [osu! Beatmap Blueprints Available & Contest Details](https://osu.ppy.sh/home/news/2015-03-20-osu-beatmap-blueprints-available-contest)*
+
+Die **osu! beatmap blueprints** (zu Deutsch Beatmap-Vorlagen) waren wieder verwendbare Aufkleber mit verschiedenen Spielelementen wie Hit-Circles, Slider, Approach-Circles, [hit judgements](/wiki/Gameplay/Judgement/osu!)<!--TODO: translate--> und dem osu!-Logo selbst. Das Produkt wurde 2015 verkauft, damit Spieler "lebensgroße Beatmaps im echten Leben" kreieren konnten.
```
Comment (FreezyLemon) [inline wiki/Past_osu!store_items/de.md:85]:
```suggestion
Die **osu! beatmap blueprints** (zu Deutsch Beatmap-Vorlagen) waren wiederverwendbare Aufkleber mit verschiedenen Spielelementen wie Hit-Circles, Slider, Approach-Circles, [hit judgements](/wiki/Gameplay/Judgement/osu!)<!--TODO: translate--> und dem osu!-Logo selbst. Das Produkt wurde 2015 verkauft, damit Spieler "lebensgroße Beatmaps im echten Leben" kreieren können.
```

File: wiki/Past_osu!store_items/de.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+Der **osu!mug** war eine Sonderedition einer Tasse, die mit Abbildungen von [pippi](/wiki/Mascots#pippi) geschmückt war. Das Produkt wurde 2016 zusammen mit osu!go verkauft.
+
+## Limitierte Edition der osu!-Plüschtiere
+
+![Limited edition osu! plushies](img/osu-limited-edition-plushie.jpg)
+
+*Für den Newsbeitrag, siehe: [Makeship x osu!: Limited Edition Plushies](https://osu.ppy.sh/home/news/2022-12-10-makeship-x-osu-plushies)*
+
+Die **limitierte Edition der osu!-Plüschtiere** waren ein Trio aus Plüschtieren bestehend aus [pippi](/wiki/Mascots#pippi), [Taikonator](/wiki/Mascots#taikonator) und dem [osu!-Cookie](/wiki/Client/Interface/Cookie). Die Produkte konnten im Frühjahr 2023 im Rahmen von Vorbestellungen erworben werden und wurden in Zusammenarbeit mit der Crowdfunding-Plattform [Makeship](https://www.makeship.com/) erstellt.
```
Comment (FreezyLemon) [inline wiki/Past_osu!store_items/de.md:125]:
```suggestion
Die **limitierte Edition der osu!-Plüschtiere** war ein Trio aus Plüschtieren bestehend aus [pippi](/wiki/Mascots#pippi), [Taikonator](/wiki/Mascots#taikonator) und dem [osu!-Cookie](/wiki/Client/Interface/Cookie). Die Produkte konnten im Frühjahr 2023 im Rahmen von Vorbestellungen erworben werden und wurden in Zusammenarbeit mit der Crowdfunding-Plattform [Makeship](https://www.makeship.com/) hergestellt.
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
Few small mistakes, looks good otherwise

File: wiki/Past_osu!store_items/de.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+Das Produkt wurde in zwei Ausführungen angeboten: Das originale "[osu!tablet v1](https://www.youtube.com/watch?v=27RkPY5lWBw)", welches zuerst 2013 hergestellt wurde. Das verbesserte [osu!tablet v2](/wiki/Guides/Tablet_purchase#osu!store) kam ab 2016 auf den Markt, bevor es im Jahre 2017 im Rahmen einer [Anpassung der Entwicklungsschwerpunkte](https://twitter.com/ppy/status/846190076853870592) eingestellt wurde.
+
+## osu! beatmap blueprints
+
+![osu! beatmap blueprints](img/osu-beatmap-blueprints.jpg)
+
+*Für den Newsbeitrag, siehe: [osu! Beatmap Blueprints Available & Contest Details](https://osu.ppy.sh/home/news/2015-03-20-osu-beatmap-blueprints-available-contest)*
+
+Die **osu! beatmap blueprints** (zu Deutsch Beatmap-Vorlagen) waren wiederverwendbare Aufkleber mit verschiedenen Spielelementen wie Hit-Circles, Slider, Approach-Circles, [hit judgements](/wiki/Gameplay/Judgement/osu!)<!--TODO: translate--> und dem osu!-Logo selbst. Das Produkt wurde 2015 verkauft, damit Spieler "lebensgroße Beatmaps im echten Leben" kreieren können.
```
Comment (The-Last-Cookie) [inline wiki/Past_osu!store_items/de.md:85]:
```suggestion
Die **osu! beatmap blueprints** (zu Deutsch Beatmap-Vorlagen) waren wiederverwendbare Aufkleber mit verschiedenen Spielelementen wie Hit-Circles, Slider, Approach-Circles, [Trefferbeurteilungen](/wiki/Gameplay/Judgement/osu!) und dem osu!-Logo selbst. Das Produkt wurde 2015 verkauft, damit Spieler "lebensgroße Beatmaps im echten Leben" kreieren können.
```

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**